cloud-sql/mysql/activerecord/config/database.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
+  <<: *default
+  database: db/development.sqlite3
```

Comment:
Is this correct? Shouldn't the development be using the database connection via the proxy?

Reply:
Totally up to you. I set it up to use sqlite3 for dev and cloudsql-mysql for production. The README instructions account for this. Happy to change it if you'd prefer.

Reply:
I think the proxy may be preferred, as it will help users understand how to configure to connect from non-appengine locations.